### ISSUE TITLE
Strange cast Iron railing side texture

### ISSUE THREAD

**Issue by WolfieWaffle** [BrassGoggledCoders/SteamCraft2#215]
I'm pretty sure this isn't supposed to happen:
![2015-06-13_14 43 37](https://cloud.githubusercontent.com/assets/8770485/8145949/3bbf0f5e-11db-11e5-9dce-786cc3ea3d64.png)

I don't have any texture packs installed, but it might be just my computer. Either way I thought I'd just put in here in case.

**Comment by warlordjones** [comment]
No...that's not mean to happen! I'll look into that.

**Comment by WolfieWaffle** [comment]
UPDATE: It seems this is also a mod incompatibility... I took out all my mods and it got fixed...

Now for the long process of adding back mods individually XD

**Comment by WolfieWaffle** [comment]
Another update: I found which mod it is. Modular Powersuits. I installed it plus Numina, it's dependency, and this happened:
![2015-06-13_15 51 56](https://cloud.githubusercontent.com/assets/8770485/8146113/6f35fc04-11e4-11e5-94d0-dab7e0281a6a.png)
I couldn't get the models to go transparent again, but at least this is isolated.

I know it's not Numina and is Powersuits because I removed all mods besides Boilerplate, SC2, and Numina. No error. Added MPS, and it happened. I hope that helps, I don't know how to fix incompatibilities, but now we know what mod is causing it. I guess make an issue on their GitHub, since I think it's probably on their side since they use some fancy rendering. I don't really know though, that's just speculation :P
